book/website/collaboration/new-community/new-recruitment.md

```diff
@@ -0,0 +1,524 @@
+# Inclusive recruitment strategies to attract diverse job applicants
+
+## Mission statement
+
+This chapter focuses on recruitment, providing practical strategies, resources and processes to reach a diverse pool of job applicants.
+It does not cover processes after the closing of the job advert such as sifting, shortlisting or interviewing.
+The intended audience is hiring managers and/or team members involved in the recruitment process, and not Human Resources (HR) specialists or dedicated recruiters. These recommendations are intended to be used alongside existing recruitment processes or guidelines within your respective organisation.
+
+While our examples will often draw from data science scenarios, the principles and strategies discussed are applicable across various tech roles and sectors.
+
+This document contains a suggested set of actions and timeframes, which may vary across organisations, as hiring processes are different.
+
+Whilst this document focuses on suggestions to attract diverse candidates, you must be aware that there will be different legal obligations around hiring based on certain characteristics depending on the country you’re hiring in. For example, the UK has the [Equality Act 2010](https://www.gov.uk/discrimination-your-rights) in place that legally protects people from discrimination in the workplace and in wider society. 
+
+There are nine protected characteristics which are legally protected under the Equality Act 2010. 
+
+These are (in no particular order): 
+* age 
+* gender reassignment
+* being married or in a civil partnership
+* [being pregnant](https://www.gov.uk/working-when-pregnant-your-rights) or on maternity leave
+* [disability](https://www.gov.uk/definition-of-disability-under-equality-act-2010)
+* race including colour, nationality, ethnic or national origin
+* religion or belief
+* sex
+* sexual orientation
+
+In some circumstances, you can use protected characteristics to help a disadvantaged or an underrepresented group during the recruitment process, for example by making reasonable adjustments for disability. This can help your organisation to be more diverse and representative. However, it is important to know what is possible and what could be against the law. Your action must not cause 'detriment' to another protected group (where someone is treated worse or discriminated against relative to someone else).
+
+Although you may want to make an impact to progress your team/organisation’s diversity, for example by  bringing more underrepresented groups from a STEM background into the organisation, it does not mean you can discount other candidates that meet the essential criteria, simply based on characteristics such as gender and/or age. This would be classed as unlawful discrimination. 
```

Comment:
```suggestion
Although you may want to make an impact to progress your team/organisation’s diversity, for example by bringing more underrepresented groups from a STEM background into the organisation, it does not mean you can discount other candidates that meet the essential criteria, simply based on characteristics such as gender and/or age. This would be classed as unlawful discrimination. 
```

Reply:
Outdated by another commit! Thank you!